test/unit/custom_spec.js

```diff
@@ -0,0 +1,99 @@
+import { buildGetDocumentParams } from './test_utils';
+import { DOMCanvasFactory } from '../../src/display/dom_utils';
+import { getDocument } from '../../src/display/api';
+import { isNodeJS } from '../../src/shared/util';
+import { PDFJS } from '../../src/display/global';
+
+function getTopLeftPixel(canvasContext) {
+  var imgData = canvasContext.getImageData(0, 0, 1, 1);
+  var r = imgData.data[0];
+  var g = imgData.data[1];
+  var b = imgData.data[2];
+  var a = imgData.data[3];
+  return { r, g, b, a};
```

Comment:
I'm not sure that the temporary variables adds much value here, let's just make it:
```javascript
  return {
    r: imgData.data[0],
    g: imgData.data[1],
    b: imgData.data[2],
    alpha: imgData.data[3],
  };
```